UPGRADE.md

```diff
@@ -7,6 +7,14 @@ When upgrading Phragile to a newer version **always** run
 
 after updating the files in the root directory of your application.
 
+## From 2.0.0 release to 3.0.0
```

Comment:
Stupid me, I just saw that you already wrote a note for this case in the UPGRADE.md and it just wasn't in this branch yet :)
Updated!